### ISSUE TITLE
Setting a new Template : existing cards are still linked to the old one

### ISSUE THREAD

**Issue by Gulix** [Gulix/geckos#34]
How to reproduce :

Open the demo file.
Replace the template with a void one (should work also with a complete one)

```
{
  "fields": [
    { "name": "name", "label": "Name", "default": "Morg N Thorg" }
  ],
  "canvasFields": [

  ],
  "canvasBackground": "#FF0000",
  "canvasWidth": 536,
  "canvasHeight": 750
}
```

Set the template.

The old card is still in the list, and the fields are the ones from the previous template.
2 solutions : 
- keep the card and update it to the new fields. If some have the same name, values is kept.
- remove all the old cards

Maybe related to #23

**Comment by Gulix** [comment]
Need to see how each type of field is managed when updating template, especially when the original and new one are from different types.

**Comment by Gulix** [comment]
The Image Input field is not correctly updated.

**Comment by Gulix** [comment]
Partially fixed with the Issue #50 
Still need to see how to manage fields that change their type (Options to Text, Text to Options, Image to Text, ...)

**Comment by Gulix** [comment]
The lasting elements of this issue are transfered to Issue #66.
